lib/tpl/dokuwiki/css/_footnotes.css

```diff
@@ -26,6 +33,11 @@ div.insitu-footnote {
 .dokuwiki div.footnotes div.fn div.content {
     display: inline;
 }
-.dokuwiki div.footnotes div.fn sup a.fn_bot {
+.dokuwiki div.footnotes div.fn a.fn_bot {
     font-weight: bold;
+	vertical-align: super;
```

Comment:
Didn't you mention that footnotes IDs should not be superscripted at the bottom?

Reply:
Indeed, and the first thing i would do in _my_ template is make sure that they won't appear this way :-) But I understand that this is how users of the default template are used to things looking, so I don't want to change the "look and feel" needlessly.

I attempted to replicate the original look and feel in CSS as good as possible. Here are two screenshots:

![image](https://github.com/user-attachments/assets/d90496b9-449b-40ca-87c0-be01ab3200c8)

The top one is with the new design, the bottom is the old one. There are minor differences (the inline number are more consistent with the text), but it tries to be as true to the original as possible.

Hm, maybe I should be braver there and just go for the way that I consider the "best practice" here ;-)